### DISCUSSION TITLE
Azure DevOps deployment getting 404 error

### DISCUSSION THREAD

**Discussion by BigTP25** [Azure-Player/azure.datafactory.tools#345] [Q&A]
I am getting a 404 error when deploying to another ADF on a different subscription.  If the ADF in my UAT environment doesn't exist, it will recreate it but runs into some problems because I have a SHIR and its linked to this UAT ADF environment...so blocker.  If the UAT exists, it says "The remote server returned an error: (404) Not Found."  Anyone have this problem? Kinda stuck...

**Comment by BigTP25** [comment]
I am getting this issue again...I can't remember I how handled it before.

**Reply by NowinskiK** [reply]
Did you raise it in GitHub before?

**Comment by BigTP25** [comment]
Yes I did

**Comment by jsebpi** [comment]
We were having the same issue. I just disabled the Incremental option in our Release Task and it finally worked. 

I'm wondering if the issue may be related to the Get-GlobalParam function or something similar since it makes an API call and we're getting 404 in response with Incremental turned on.

**Comment by BigTP25** [comment]
I fixed things because I had a small error in my config file.  But the
error is not very helpful.

**Reply by NowinskiK** [reply]
I can't help when people do not provide enough details allowing me to reproduce the error scenario. If you still feel you're facing the issue due to an error in this module - create a new issue and provide enough details.

**Comment by NowinskiK** [comment]
@BigTP25, can you advise what error you've got in your config?

**Comment by NicoSaar** [comment]
I'm actually having the same issue & my config file is valid.
Disabling the Incremental option fixed the error for the moment.

**Reply by NowinskiK** [reply]
I can't help when people do not provide enough details allowing me to reproduce the error scenario. If you still feel you're facing the issue due to an error in this module - create a new issue and provide enough details.

**Reply by NicoSaar** [reply]
The use of the Azure Player is quite new to me, so I will check again to what extent the configuration of the pipeline and the existing files may be wrong.
The deployment of the Azure Data Factory artifacts works after all if the flag for incremental deployment is not set. As soon as I know more details, I would create a new issue and of course provide detailed information.
Thank you very much for your work and support!

**Comment by BigTP25** [comment]
@Kamil

I had a small error in the config that was pointing to the wrong resource
group.

**Reply by NowinskiK** [reply]
Hence, there was nothing related to Incremental Deployment feature, is that correct?

**Comment by BigTP25** [comment]
I don't think so, not for me at least...

**Comment by BigTP25** [comment]
I appreciate the responses. For me, I double checked my configs for the
environment or "stage" I was deploying to and found an issue with the
resource group I was saying a particular resource should reside. I fixed
that and everything deployed fine for me.  I mean the 404 does make sense
when I look at it from that POV... but it is kind of a generic error
returned that doesn't point you in any direction. It would be nice if there
was some sort of check on the config value stipulated and what is specified
in the task and returned a better error. But ultimately, for me, it was
user error and I appreciate the hard work and continued support. Thanks for
your help!
